<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frames</class>
 <widget class="QDialog" name="Frames">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>744</width>
    <height>435</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>744</width>
    <height>435</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AvisionR - Frames</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(255, 255, 255);
background-color: #262626;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Frame">
     <property name="styleSheet">
      <string notr="true">border: 1px solid #686868;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Brand">
     <property name="styleSheet">
      <string notr="true">border: 1px solid #686868;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QLineEdit" name="lineEdit_Model">
     <property name="styleSheet">
      <string notr="true">border: 1px solid #686868;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QTableView" name="tableView_frames">
     <property name="styleSheet">
      <string notr="true">QHeaderView::section {
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #616161, stop: 0.5 #505050,
                                       stop: 0.6 #434343, stop:1 #656565);
     color: white;
     padding-left: 4px;
     border: 1px solid #6c6c6c;

}

QTableCornerButton::section 
{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #616161, stop: 0.5 #505050,
                                       stop: 0.6 #434343, stop:1 #656565);
     color: white;
     padding-left: 4px;
     border: 1px solid #6c6c6c;
}

color:rgb(255, 255, 255);
background-color: #262626;
</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCommandLinkButton" name="commandLinkButton_New">
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>New</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>345</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QCommandLinkButton" name="commandLinkButton_Done">
     <property name="maximumSize">
      <size>
       <width>91</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QCommandLinkButton" name="commandLinkButton_Clear">
     <property name="maximumSize">
      <size>
       <width>91</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QCommandLinkButton" name="commandLinkButton_Search">
     <property name="maximumSize">
      <size>
       <width>91</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_Brand</sender>
   <signal>returnPressed()</signal>
   <receiver>commandLinkButton_Search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Frame</sender>
   <signal>returnPressed()</signal>
   <receiver>commandLinkButton_Search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Model</sender>
   <signal>returnPressed()</signal>
   <receiver>commandLinkButton_Search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
